### PR TITLE
[flutter_local_notifications_linux] bump linux plugin for release

### PR DESCRIPTION
**Pull request by MaikuB**

### CODE REVIEW

File: flutter_local_notifications_linux/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+## [0.2.0]
+
+* Fixed issue when an app using the plugin is built on the web by using conditional imports
+* Changed the logic where notification IDs are saved so that `$XDG_RUNTIME_DIR` environment variable is not set but `$TMPDIR` is set, then they are saved to a file within the `/$TMPDIR/APP_NAME/USER_ID/SESSION_ID` directory. If `$TMPDIR` is not set then, it would save to `/tmp/APP_NAME/USER_ID/SESSION_ID`
```
Comment (MaikuB) [inline flutter_local_notifications_linux/CHANGELOG.md:4]: @proninyaroslav would you be able to help check if I got the description right?

Comment (proninyaroslav) [reply flutter_local_notifications_linux/CHANGELOG.md:4]: Yes that's right

Comment (proninyaroslav) [reply flutter_local_notifications_linux/CHANGELOG.md:4]: You can also add about the double initialization fix

Comment (MaikuB) [reply flutter_local_notifications_linux/CHANGELOG.md:4]: ah yes, I missed that. thanks :)

**Pull request merged**